core/src/miner/miner.rs

```diff
@@ -328,6 +328,9 @@ impl Miner {
         };
 
         assert!(self.engine.seals_internally(), "If a signer is not prepared, prepare_block should not be called");
+        // FIXME this sleeping is for the experimental branch. This should not be included in the master branch.
```

Comment:
In master, instead of this, we will fix the tendermint core logic?

Reply:
We don't need it.
Since network latency, a Foundry used in a network does not need this code.